solr/core/src/test/org/apache/solr/handler/admin/api/ReloadCoreAPITest.java

```diff
@@ -57,7 +57,6 @@ public void setUp() throws Exception {
   public void testValidReloadCoreAPIResponse() throws Exception {
     SolrJerseyResponse response = reloadCoreAPI.reloadCore(coreName, new ReloadCoreRequestBody());
     assertEquals(0, response.responseHeader.status);
-    assertNotNull(response.responseHeader.qTime);
```

Comment:
Why did this test change?

Reply:
According to error-prone:

> error: [ImpossibleNullComparison] This value cannot be null, and comparing it to null may be misleading.

This is because primitive data types have always a default value. Resolving this to focus on the root discussion.